src/drivers/distance_sensor/vl53lxx/vl53lxx.cpp

```diff
@@ -69,37 +69,39 @@
 #include <board_config.h>
 
 /* Configuration Constants */
-#define VL53LXX_BUS_DEFAULT PX4_I2C_BUS_EXPANSION
+#define VL53LXX_BUS_DEFAULT				PX4_I2C_BUS_EXPANSION
 
-#define VL53LXX_BASEADDR 0b0101001 // 7-bit address
-#define VL53LXX_DEVICE_PATH "/dev/vl53lxx"
+#define VL53LXX_BASEADDR				0x52
```

Comment:
You are using wrong BASEADDR, you should use unshifted address, the original one is right. 0b0101001 or 0x29.

Reply:
Thanks @bys1123 , I got that from the datasheet.  I'll put it back to 0x29, looks like they messed up the datasheet:

![image](https://user-images.githubusercontent.com/2497951/57974719-2aa5f100-797a-11e9-8a16-4d49d9a0729b.png)

Reply:
Actually, you can't say it's a mistake, see [here](https://github.com/sparkfun/SparkFun_VL53L1X_Arduino_Library/issues/7#issuecomment-389286501) . 
I was confused too, when I see this driver first time.